### PR TITLE
Rewritten some bits

### PR DESCRIPTION
**Pull request by emilyb7**
Hey @shiryz, here's a redraft of your awesome workshop!

I fixed some typos, but also tried to make some of the instructions a bit clearer. I hope it has been useful. Some stuff is just really hard to explain! I've probably made some new typos in the process now...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,67 +1,71 @@
...
 | ------- | --------- | ---- | -------------------- | ------- |
 
-your challenge is to construct a query that returns the book_id, book_name, max_reservation_time
-of books that can be reserved for time that is greater than the avg reservation time
-in the book's library.
+your challenge is to construct a query that returns the following columns:
+* `book_id`, 
+* `book_name`, 
+* `max_reservation_time`,
+* `book_id`,
```
Comment (shiryz) [inline README.md]: This is repeated twice.

File: README.md [hunk 2]
```diff
@@ -1,67 +1,71 @@
...
-your challenge is to construct a query that returns the book_id, book_name, max_reservation_time
-of books that can be reserved for time that is greater than the avg reservation time
-in the book's library.
+your challenge is to construct a query that returns the following columns:
+* `book_id`, 
+* `book_name`, 
+* `max_reservation_time`,
+* `book_id`, 
+* `book_name max_reservation_time`
+**AND** to return only the books that can be reserved for a time greater than the **average** reservation time for all books at the given library.
```
Comment (shiryz) [inline README.md]: How should we phrase it, in order to stress that we only want the library the book is in and not all the libraries?

File: README.md (hunk 1 above)
Comment (emilyb7) [inline README.md]: oops

File: README.md (hunk 2 above)
Comment (emilyb7) [inline README.md]: that's why I put "given library", but if you think that's not clear enough we could try "... for all the books at the library this book is in" 
what do you think?

Comment (shiryz) [inline README.md]: Yeah I think that sounds clearer. :smile:

File: README.md (hunk 1 above)
Comment (emilyb7) [inline README.md]: fixed!

File: README.md (hunk 2 above)
Comment (emilyb7) [inline README.md]: fixed!

**Comment by shiryz** [comment]
Perfect comments and fixes!

**Pull request merged**